components/wifi/esp8266-driver/ESP8266/ESP8266.cpp

```diff
@@ -1201,4 +1201,29 @@ nsapi_connection_status_t ESP8266::connection_status() const
 {
     return _conn_status;
 }
+
+bool ESP8266::set_country_code_policy(bool track_ap, const char *country_code, int channel_start, int channels)
+{
+    int _track_ap = track_ap ? 0 : 1;
```

Comment:
[minor] isn't an underscore reserved for private class members?

Reply:
Fixed